tests/services/fontgen_test.py

```
@@ -0,0 +1,276 @@
+import shutil
+import subprocess
+import unittest
+
+try:
+    import numpy  # noqa: F401
+    import freetype  # noqa: F401
+    from polyhost.services import fontgen
+    from polyhost.services.fontgen import RenderOptions
+    from polyhost.services import fontgen_dither as fd
+    _ERR = None
+except Exception as e:  # pragma: no cover
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)